Automattic/wp-calypso
# Layout: makes wide layout full width.

#### Changes proposed in this Pull Request

* Makes `is-wide-layout` full width
* Removes all instances of `is-wide-layout` in favor of `wideLayout={ true }`
* Removes `wideLayout={ true }` from `https://wordpress.com/me` page. This page and `https://wordpress.com/me/account` are the ones where the wide layout is questionable. I opted for narrow layout in "My Profile" but kept the wide layout in "Account Settings" 

#### Testing instructions



* Navigate around Calypso and check if there other cases where we should remove the wide layout too.

Before | After
-------|------
![](https://cln.sh/YLysUA+) | ![](https://cln.sh/ppw4dh+)

Before (My Profile) | After (My Profile)
-------|------
![](https://cln.sh/7JggXo+) | ![](https://cln.sh/eOSTOm+)



Related to #51096
